project/Dependencies.scala

```diff
@@ -33,11 +31,12 @@ object Dependencies {
 
   val sslConfigVersion = "0.4.1"
 
+  val scalaTestVersion = "3.1.1"
+  val scalaCheckVersion = "1.14.3"
+
   val Versions = Seq(
     crossScalaVersions := Seq(scala212Version, scala213Version),
     scalaVersion := System.getProperty("akka.build.scalaVersion", crossScalaVersions.value.head),
-    scalaCheckVersion := sys.props.get("akka.build.scalaCheckVersion").getOrElse("1.14.3"),
```

Comment:
Why did we have it as a setting/sys prop before, is it used somewhere?

Reply:
Looks like it was used before to cross-build between scala 2.11 and 2.12 (https://github.com/akka/akka/commit/5dc25fbb442d302c90f19853c93b6021542b5aea)